lara-typescript/src/lib/plugin-context.ts

```diff
@@ -138,7 +138,7 @@ const log = (context: IPluginContextOptions, logData: string | ILogData): void =
 };
 
 const fetchPluginEventLogData = (context: IPluginContextOptions) => {
```

Comment:
It is too late to change this now. But it is code like this, which is why using Options doesn't seem right. This object is not just a set of options to configure the context. It is actually the internal object used by the plugin api implementation.

Reply:
That shouldn't be using options but rather the context.  Its easy to fix - I'll do that now.